src/DDTrace/OpenTelemetry/Context.php

```diff
@@ -58,8 +61,13 @@ public static function setStorage(ContextStorageInterface $storage): void
      */
     public static function storage(): ContextStorageInterface
     {
-        /** @psalm-suppress RedundantPropertyInitializationCheck */
-        return self::$storage ??= new ContextStorage();
+        if (self::$storageClass === '') {
+            self::$storageClass = class_exists('\OpenTelemetry\Context\FiberBoundContextStorageExecutionAwareBC')
+                ? '\OpenTelemetry\Context\FiberBoundContextStorageExecutionAwareBC' // v1.1+
+                : '\OpenTelemetry\Context\ContextStorage';
```

Comment:
```suggestion
            self::$storageClass = class_exists('OpenTelemetry\Context\FiberBoundContextStorageExecutionAwareBC')
                ? 'OpenTelemetry\Context\FiberBoundContextStorageExecutionAwareBC' // v1.1+
                : 'OpenTelemetry\Context\ContextStorage';
```
Removing the leading slashes should optimize a little too.